test/diameter_Rf_SUITE.erl

```diff
@@ -59,6 +59,8 @@
 		    #{handler => 'ergw_aaa_rf'}},
 	  apps =>
 	    #{default =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init => [#{service => <<"Default">>}],
 		      authenticate => [],
 		      authorize => [],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok